docs/docs/01-introduction.md

```diff
@@ -18,18 +18,19 @@
 
 ### What is Snowpack?
 
-**Snowpack is a faster build tool for modern web apps.** Snowpack leverages [ESM imports](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import) in your application to remove unnecessary bundling work during development. The end result is a build tool that starts up instantly and wastes no time rebuilding on every change. See changes reflected in the browser instantly.
+**Snowpack  is a modern, lightweight toolchain for web application development.** Traditional dev bundlers -- like Webpack or Parcel -- need to rebuild & rebundle entire chunks of your application every time you save a single file. This introduces lag between changing a file and seeing those changes reflected in the browser, sometimes several seconds long in larger applications.
```

Comment:
```suggestion
**Snowpack is a modern, lightweight toolchain for web application development.** Traditional dev bundlers like webpack or Parcel need to rebuild & rebundle entire chunks of your application every time you save a single file. This introduces lag between changing a file and seeing those changes reflected in the browser, sometimes as slow as several seconds.
```

nit:
- no em-dash needed since you’re not really breaking your flow of thought? But it doesn’t hurt to have if you feel it breaks the sentence up better
- webpack is lowercase
- IMO that end sentence is one of the most important things to understand about Snowpack; I wanted to give it a bit more punch